code/modules/modular_computers/file_system/programs/alarm.dm

```diff
@@ -72,15 +72,23 @@
 /datum/computer_file/program/alarm_monitor/proc/cancelAlarm(class, area/A, obj/origin)
 	var/list/L = alarms[class]
 	var/cleared = 0
+	var/arealevelalarm = 0 // set to 1 for alarms that set/clear whole areas
```

Comment:
Use `TRUE`/`FALSE` defines.

Reply:
Sure, I guess I was mimicing the nearby var/cleared which seems to be a numeric boolean
If I can work out how to update the pull request anyway.